membership_activity/tests/test_membership_activity.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Onestein
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from datetime import datetime
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestMembershipActivity(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.partner = self.env["res.partner"].create({"name": "Test Member"})
+        self.project = self.env["project.project"].create({"name": "Test project"})
+        self.membership_activity_type = self.env["membership.activity.type"].create(
+            {"name": "Testing"}
+        )
+        self.membership_activity = self.env["membership.activity"].create(
+            {
+                "partner_id": self.partner.id,
+                "project_id": self.project.id,
+                "date": fields.Date.today(),
+                "type_id": self.membership_activity_type.id,
+            }
+        )
+
+    def test_name_get(self):
+        self.assertTrue(self.membership_activity.name_get())
+
+    def test_reconcile_partner(self):
+        self.membership_activity.partner_id = False
+        self.membership_activity.reconcile_partner()
```

Comment:
What are we testing here? I guess you just want to increase coverage with this, please add a comment if so.